modules/distr-tracing-tempo-install-web-console.adoc

```diff
@@ -0,0 +1,139 @@
++
+[IMPORTANT]
+====
+This installs the Operator with the default presets:
+
+* *Update channel* -> *stable*
+* *Installation mode* -> *All namespaces on the cluster*
+* *Installed Namespace* -> *openshift-tempo-operator*
+* *Update approval* -> *Automatic*
+====
+
+.. In the *Details* tab of the installed Operator page, under *ClusterServiceVersion details*, verify that the installation *Status* is *Succeeded*.
+
+. Create a Secret for your object storage bucket.
++
+You can do this in the *Administrator* view of the OpenShift web console: go to *Workloads* -> *Secrets* -> *Create* -> *From YAML*.
++
+--
+include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
+--
++
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+
+. Create an OpenShift project of your choice for the *TempoStack* instance that you will create in the next step.
++
+You can do this in the *Administrator* view of the OpenShift web console: go to *Home* -> *Projects* -> *Create Project*.
+
+
+. Create a *TempoStack* instance.
++
+NOTE: You can create multiple *TempoStack* instances in separate projects on the same cluster.
++
+You can create a *TempoStack* instance as follows:
+
+.. Go to *Operators* -> *Installed Operators*.
++
+[TIP]
+====
+You can use the *Project:* dropdown list to select the OpenShift project that you created for the *TempoStack* instance in the previous step.
+====
+
+.. Select *TempoStack* -> *Create TempoStack* -> *YAML view*.
+
+.. In the *YAML view*, customize the `TempoStack` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: sample
+  namespace: <openshift_project_of_tempostack_instance>
+spec:
+  storageSize: 1Gi
+  storage:
+    secret:
+      name: <secret-name> <1>
+      type: <secret-provider> <2>
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+<1> The value of `name` in the `metadata` of the Secret.
+<2> The accepted values are `azure` for Azure Blob Storage, `gcs` for Google Cloud Storage, and `s3` for Amazon S3 or MinIO or Red Hat OpenShift Data Foundation.
++
+.TempoStack CR for AWS S3 and MinIO storage
+====
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simplest
+  namespace: <openshift_project_of_tempostack_instance>
+spec:
+  storageSize: 1Gi
+  storage:
+    secret:
+      name: minio-test
+      type: s3
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          route:
+            termination: edge
+          type: route
+----
+The stack deployed in this example is configured to receive Jaeger Thrift over HTTP and OpenTelemetry Protocol (OTLP), which permits visualizing the data with the Jaeger UI.
+====
+
+.. Select *Create*.
```

Comment:
```suggestion
.. Click *Create*.
```

Reply:
Thank you. I prefer to use `select` over `click` for reasons of accessibility.